komodoenv/__main__.py

```diff
@@ -0,0 +1,153 @@
+from __future__ import print_function
+import os
+import sys
+import argparse
+import logging
+from shutil import rmtree
+from pathlib import Path
+from komodoenv.creator import Creator
+from komodoenv.statfs import is_nfs
+from colors import red, yellow, strip_color
+
+
+def get_text(release_path):
```

Comment:
get what text? Could you qualify what is to be returned here?

Reply:
Made the function name better (maybe?).